instances.tf

```diff
@@ -1,13 +1,31 @@
 module "BastionHostLinux" {
-  source    = "github.com/UKHomeOffice/connectivity-tester-tf"
-  user_data = "CHECK_self=127.0.0.1:80 CHECK_google=google.com:80 CHECK_googletls=google.com:443 LISTEN_http=0.0.0.0:80"
-  subnet_id = "${aws_subnet.OPSSubnet.id}"
+  source          = "github.com/UKHomeOffice/connectivity-tester-tf"
+  user_data       = "LISTEN_http=0.0.0.0:22"
```

Comment:
listening on port 22 will probably crash the connectivity tester, clashing with ssh, can't we try connecting to some things in the apps vpc?